source/routes/delete/bullet.js

```diff
@@ -0,0 +1,21 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require("pouchdb-find"));
+const db = new PouchDB('db');
+
+// Send in a valid json containing just the id and it will be expunged from the database
+module.exports = {
+    "/delete/bullet": {
+        methods: ['delete'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7402063)
```suggestion
        methods: ["delete"],
```